torch/_inductor/pattern_matcher.py

```diff
@@ -833,13 +833,25 @@ def percolate_tags(node, recompute_tag, input_stops):
             replacement = Replacer(replacement_graph).run(*args)
             if isinstance(replacement, torch.fx.Node):
                 replacement = [replacement]
-            assert len(replacement) == len(output_nodes)
-            for old, new in zip(output_nodes, replacement):
+
+            def maybe_getitem(node):
+                if node.op != "call_function":
+                    return None
+                if str(node.target) != "<built-in function getitem>":
```

Comment:
you can just do `node.target != operator.getitem` here

Reply:
Do you know where `operator` is off the top of your head? I remember not being able to find `operator.getitem`

Reply:
found it, it's just `import operator` (facepalm)